Public/Src/IDE/Generator.Old/BuildXL.Ide.Generator.Old.dsc

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.

```

Comment:
safe to ignore everything under `Public/Src/IDE/Generator.Old` (since it is the verbatim copy of what used to be `Public/Src/IDE/Generator`